packages/beacon-node/src/node/nodejs.ts

```diff
@@ -9,14 +9,14 @@ import {Api} from "@lodestar/api";
 import {BeaconStateAllForks} from "@lodestar/state-transition";
 import {ProcessShutdownCallback} from "@lodestar/validator";
 
+import {initializeExecutionBuilder, initializeExecutionEngine} from "@lodestar/engine-api-client";
```

Comment:
You might break this PR up into two parts.
Just move the queue to utils and move execution to this new package.
We can get these merged more quickly if they are just moving some directories around.

Also agree with @g11tech 's naming suggestion re the new package: `@lodestar/execution`